decoder/completion_resolve.go

```diff
@@ -0,0 +1,13 @@
+package decoder
+
+import (
+	"context"
+)
+
+func (d *Decoder) ResolveCandidate(ctx context.Context, unresolvedCandidate UnresolvedCandidate) (*ResolvedCandidate, error) {
+	if resolveFunc, ok := d.ctx.CompletionResolveHooks[unresolvedCandidate.ResolveHook.Name]; ok {
```

Comment:
Do you mind adding a `nil` check here for `unresolvedCandidate.ResolveHook` and return early, to avoid crashes?

Reply:
@dbanck I think this still remains unresolved - and would lead to a crash if passed `unresolvedCandidate.ResolveHook` is `nil`.

Reply:
Actually - ignore me - somehow I was looking at an old revision! 🙈